README.md

```diff
@@ -21,6 +21,23 @@ the content.
 The RukPak project is community driven and is part of the broader Kubernetes ecosystem. New contributors are welcome and
 highly encouraged. See the [contributing guidelines](CONTRIBUTING.md) to get started.
 
+## Quickstart
+
+The RukPak project consists of a series of controllers, known as provisioners, that install and manage content on a
+Kubernetes cluster. See below for a more detailed look into the APIs that RukPak provides.
+
+The provisioner currently implemented and bundled with RukPak is known as the plain provisioner. To get started with
+this provisioner, see [the quickstart section](./internal/provisioner/plain/README.md#Running-locally) of the plain
```

Comment:
Might be worth specifying that this quickstart is for running locally vs. on a cluster.